config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-discovery.yaml

```diff
@@ -65,20 +65,20 @@ periodics:
   extra_refs:
   - org: kubernetes
     repo: kubernetes
-    base_ref: release-1.28
+    base_ref: release-1.29
```

Comment:
this branch will not exist until nov 21st
https://github.com/kubernetes/sig-release/tree/master/releases/release-1.29